EndavaTechCourseBankApp.Application/Commands/UpdateUser/UpdateUserHandler.cs

```diff
@@ -0,0 +1,44 @@
+﻿using EndavaTechCourseBankApp.Domain.Models;
+using EndavaTechCourseBankApp.Infrastructure.Persistence;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EndavaTechCourseBankApp.Application.Commands.UpdateUser
+{
+    public class UpdateUserHandler : IRequestHandler<UpdateUserCommand, CommandStatus>
+    {
+        private readonly ApplicationDbContext context;
+
+        public UpdateUserHandler(ApplicationDbContext context)
+        {
+            ArgumentNullException.ThrowIfNull(context);
+            this.context = context;
+        }
+
+        public async Task<CommandStatus> Handle(UpdateUserCommand request, CancellationToken cancellationToken)
+        {
+            var user = await context.Users.FirstOrDefaultAsync(u => u.Email == request.Email);
+            
+            if(user is null)
+            {
+                return new CommandStatus { IsSuccessful = false };
```

Comment:
use the static method for failed commands : CommandStatus.Failed(string message)